kythe/go/extractors/config/smoke/smoke.go

```diff
@@ -133,16 +134,21 @@ func (g harness) TestRepo(repo string) (Result, error) {
 	}, nil
 }
 
-func (g harness) filenamesFromRepo(repoURL string) (map[string]bool, error) {
-	repoName := pathTail(repoURL)
+func (g harness) filenamesFromRepo(repoURI string) (map[string]bool, error) {
+	repoName := pathTail(repoURI)
 
 	repoDir, err := ioutil.TempDir("", repoName)
 	if err != nil {
-		return nil, fmt.Errorf("failed to create temp dir for repo %s: %v", repoURL, err)
+		return nil, fmt.Errorf("failed to create temp dir for repo %s: %v", repoURI, err)
 	}
 	defer os.RemoveAll(repoDir)
 
-	if err = g.repoFetcher.Fetch(repoURL, repoDir); err != nil {
+	repo := config.Repo{
```

Comment:
```
	if err = g.repoFetcher.Fetch(config.Repo{
		URI:        repoURI,
		OutputPath: repoDir,
	}); err != nil {
		return nil, err
	}
```

Reply:
Ah cool for some reason I figured the newline breaking in the if statement wasn't kosher.